### PR TITLE
Add Lua

### LINKED ISSUE #250: Add LUA detection
### Some reference to the SDK, such as the developer site or a wikipedia article.

LUA: https://www.lua.org/start.html
https://luabinaries.sourceforge.net/

### SteamDB links

https://steamdb.info/app/295950
https://steamdb.info/app/234390
https://steamdb.info/app/1935460/

### Possible way to detect it

Contain any of the files: 
lua50.dll, lua51.dll, lua52.dll, lua53.dll, lua54.dll
lua5.0.dll, lua5.1.dll, lua5.2.dll, lua5.3.dll, lua5.4.dll
liblua50.so, liblua51.so, liblua52.so, liblua53.so, liblua54.so

LUA-files: *.lua

### PR DESCRIPTION
**Pull request by diedyesterdaywashere**
### SteamDB app page links to a few games using this

https://steamdb.info/app/295950
https://steamdb.info/app/234390
https://steamdb.info/app/1935460/

### Brief explanation of the change

closes #250 
Added a rule for Lua inspired by issue linked above

### CODE REVIEW

**Comment by diedyesterdaywashere** [comment]
@Lifeismana maybe you know whats going wrong? because i cant figure out, maybe i am just sleepy idk

**Comment by Lifeismana** [comment]
Might be worth it to also look for `.lua` files

**Comment by diedyesterdaywashere** [comment]
true but i just went with the issue that was made before, so ye, i can make just .lua rule tho

File: rules.ini [hunk 1]
```diff
@@ -170,6 +170,7 @@ Intel_OID = (?:^|/)OpenImageDenoise\.dll$
 Intel_XeSS = (?:^|/)(?:ig|lib)xess\.dll$
 iZotope = (?:^|/)iZotope\.dll$
 JVM = (?:^|/)bin/java(?:\.exe|\.dll)?$
+Lua = (?:^|/)(?:lua50\.dll|lua51\.dll|lua52\.dll|lua53\.dll|lua54\.dll|lua5.0\.dll|lua5.1\.dll|lua5.2\.dll|lua5.3\.dll|lua5.4\.dll|liblua50\.so|liblua51\.so|liblua52\.so|liblua53\.so|liblua54\.so|\.lua)$
```
Comment (Lifeismana) [inline rules.ini:173]:
```suggestion
Lua[] = (?:^|/)(?:lib)?lua5.?[01234]\.(?:dll|so)$
Lua[] = \.lua$
```
untested so might be wrong
(run the scripts and fix things if you can)

**Pull request merged**